### DISCUSSION TITLE
Error! You are running an unsupported browser page

### DISCUSSION THREAD

**Discussion by bajlek** [globaleaks/globaleaks-whistleblowing-software#4230] [General]
Hi, Is it possible to eddit warning about unsuported browser notification?

![image](https://github.com/user-attachments/assets/1799c981-632f-4e65-b33d-517a1e52a4f8)

**Comment by evilaliv3** [comment]
Hello @bajlek 

At the moment no it is not possible.

Which needs do you have?

Typically this page is seen in very very rare occasions since the platform does not really support any browser though. On which browser are you incurring in this situation ?

**Comment by bajlek** [comment]
Actually I want to edit whole text. Especially for languages purposes, but also for less information :-)

**Comment by evilaliv3** [comment]
I see. 

Actually that part of the application is shown when the application do not run so at the moment there is no way one would be able to customize or for example asses the language of the user.

But as told it will shows up only in very rare occasions.

I'm curious in which situation are you that you are currently noticing that message.

**Reply by bajlek** [reply]
So is it a part of soome js script which can I edit? Situation was with using very old Android phone and no so old phone with some browser from Samsung I think.

**Reply by evilaliv3** [reply]
At the moment it is just text inside the basic index.html file.

We suggest to not edit it manually as you may incurr in significantly tempering the app.

**Reply by bajlek** [reply]
I dont see that text 

```
<app-root class="container-fluid d-flex flex-column h-100 px-0"></app-root>

<noscript>
    <div class="js-disabled-message">
        <h1>Error! :(</h1><br>
        <div>
            Your browser is not running JavaScript that is required to use this site. <br> <br> It is a common belief that
            JavaScript and security don't work well together and for this reason we advise to use the
            <a href="https://www.torproject.org/" rel="noreferrer" target="_blank"> Tor Browser </a> , a browser designed
            for security and privacy.
        </div>
    </div>
</noscript>

<script src="js/runtime.js" type="module"></script><script src="js/polyfills.js" type="module"></script><script src="js/vendor.js" type="module"></script><script src="js/main.js" type="module"></script>
```

**Reply by evilaliv3** [reply]
You are right, in current app is in https://github.com/globaleaks/whistleblowing-software/blob/main/client/app/src/pages/app/app.component.html so  maybe for the future we may find a way to make it customable or translatable.

At the moment though you have unfortunately no easy options to do that.

**Reply by bajlek** [reply]
Thank you for your time!
